Makefile

```diff
@@ -56,7 +57,7 @@ INITRD_DIR := $(BUILD)/initrd
 
 $(INITRD_DIR)/images.tar: bin/rancheros
 	ln -sf bin/rancheros ./ros
-	for i in `./ros c images -i os-config.yml`; do docker pull $$i; done
+	for i in `./ros c images -i os-config.yml`; do [ "$(PULL_IMAGES)" != "1" ] && docker inspect $$i >/dev/null 2>&1 || docker pull $$i; done
```

Comment:
Honestly this line is really hard to read.  You have a negated condition followed by a && then ||